dashboard/app/models/user.rb

```diff
@@ -223,6 +225,19 @@ def save_email_preference
     end
   end
 
+  # Enables/disables email notifications for the parent.
+  def save_parent_email_preference
+    if student? && parent_email.present?
+      EmailPreference.upsert!(
+        email: parent_email,
+        opt_in: parent_email_preference_opt_in.downcase == "yes",
+        ip_address: parent_email_preference_request_ip,
+        source: parent_email_preference_source,
+        form_kind: nil
```

Comment:
Do we want to double check that this isn't used?

Reply:
I just did a quick pass through all the code mentioning "form_kind". It doesn't look like anything which would affect us. There are also unit tests which validate that its okay to have `nil` as a value